packages/graph/README.md

```diff
@@ -1,40 +1,83 @@
 # The HASH Graph API
 
-## Running the database
+To run the HASH Graph API make sure `cargo-make` is installed:
+
+```shell
+cargo install cargo-make
+```
+
+## Run the Graph
 
 1.  In order to set up the database, first the database has to be started:
 
 ```shell
-touch .env.local
-docker-compose -f deployment/docker-compose.yml --env-file .env up
+cargo make deployment-up
 ```
 
-Alternatively, simply run
+It's possible to recreate the database by using
 
 ```shell
-cargo make deployment-up
+cargo make recreate-db
+```
+
+Then, the Graph API can be started:
+
+```shell
+cargo run
 ```
 
-1.  (Optional) If starting the database for the first time, the tables have to be generated. From the `migration` directory, run the following command:
+## Development
+
+In order to build run the following command:
 
 ```shell
-yarn graph:recreate-db
+cargo make build
 ```
 
-or
+In order to create an optimized build, run:
 
 ```shell
-cargo make recreate-db
+cargo make --profile production build
+```
+
+Please see the list of all possible `cargo make` commands:
+
+```shell
+cargo make --list-all-steps
+```
+
+Every command line argument passed will also be forwarded to the subcommand, e.g. this will not only build the documentation but also open it in the browser:
+
+```shell
+cargo make doc --open
+```
+
+## Test the code
+
+The code base has two test suites: The unit test suite and the integration tests. To run the unit-test suite, simply run the `test` command:
+
+```shell
+cargo make test
+```
+
+For the integration tests, the database needs to be deployed [as specified here](../README.md#running-the-database). Next, the integration test suite can be started:
+
+```shell
+cargo make test-integration
 ```
 
-1.  Then, apply the migrations from the `migration` directory:
+The REST API can be tested as well. Note, that this requires the Graph to run and does not clean up the database after running:
```

Comment:
Has this changed? I don't think it did require the graph to be running

Reply:
Yes, it's required, it's needed that the binary is running in order to run `httpyac`.

I moved the README.md from `hash_graph` to the top level as this only contained one statement now.

Reply:
Before we had some code that started the binary and trapped it, it seems it was removed prior to this PR though. No worries